sdk/python/lib/test/langhost/util.py

```diff
@@ -89,6 +89,17 @@ def RegisterResource(self, request, context):
                 }
 
             self.reg_count += 1
+        else:
+            # Record the Stack's registration so that it can be the target of register_resource_outputs
```

Comment:
This was an error in the test framework that prevented us from being able to observe stack outputs in unit tests.